Resources/Prototypes/_Nuclear14/Entities/Objects/Consumable/Flora/wastelandflora.yml

```diff
@@ -312,24 +312,30 @@
 
 - type: entity
   parent: N14WastelandFlora
-  id: N14WastelandFloraWildJalapenoPepper
-  name: wild jalapeno pepper plant
-  description: A jalapeno pepper plant growing wild. It's healthy.
+  id: N14WastelandFloraWildGlowpepper #N14 change
+  name: wild glowpepper plant #N14 change
+  description: A glowpepper plant GLOWING wild. AH HA HA HA... it's healthy. #N14 change
   components:
   - type: Sprite
-    sprite: _Nuclear14/Objects/Specific/Hydroponics/jalapeno.rsi
+    sprite: _Nuclear14/Objects/Specific/Hydroponics/jalapeno.rsi #I DON'T KNOW WHERE THIS IS
```

Comment:
Found in resources/textures

Reply:
I'll remember that for later - is it good practice to rename this sprite?

Reply:
Just pointing you where they are in case you need to change them in future